app/controllers/application_controller.rb

```diff
@@ -45,7 +45,9 @@ def cors_set_headers
   end
 
   def current_season
-    @season ||= Season.current
+    #TEMP solution to find the application form
+    #@season ||= Season.current
+    @season = Season.find(2)
```

Comment:
This is a dirty workaround for something I have to solve first. I am working on that in a different branch, but I found no other way than this to reach the application form.